modules/pagination/client-react/containers/PaginationDemo.native.jsx

```diff
@@ -0,0 +1,113 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { StyleSheet, View, Text, TouchableOpacity } from 'react-native';
+import { translate } from '@restapp/i18n-client-react';
+import { styles as lookStyles, Select } from '@restapp/look-client-react-native';
+import PaginationDemoView from '../components/PaginationDemoView.native';
+import { withDataProvider } from './DataProvider';
+
+@translate('pagination')
+@withDataProvider
+class PaginationDemo extends Component {
+  static propTypes = {
+    t: PropTypes.func,
+    items: PropTypes.object,
+    loadData: PropTypes.func
+  };
+
+  state = { pagination: 'standard' };
+
+  onPaginationTypeChange = itemValue => {
+    const { loadData, items } = this.props;
+    this.setState({ pagination: itemValue }, loadData(0, items.limit));
+  };
+
+  handlePageChange = (pagination, pageNumber) => {
+    const { loadData, items } = this.props;
+    if (pagination === 'relay') {
+      loadData(items.pageInfo.endCursor, 'add');
+    } else {
+      loadData((pageNumber - 1) * items.limit, 'replace');
+    }
+  };
+
+  renderItem = ({
+    item: {
+      node: { title }
+    }
+  }) => {
+    return (
+      <TouchableOpacity style={styles.postWrapper}>
+        <Text style={styles.text}>{title}</Text>
+      </TouchableOpacity>
+    );
+  };
+
+  render() {
+    const { t, items } = this.props;
+    const { pagination } = this.state;
+    const options = [
```

Comment:
`render` is not a place for defining data.

Reply:
Fixed